packages/mobile/android/build.gradle

```diff
@@ -38,7 +38,7 @@ allprojects {
             url("$rootDir/../../../node_modules/jsc-android/dist")
         }
         maven {
-            url "$rootDir/../../../node_modules/detox/Detox-android"
+            url "$rootDir/../node_modules/detox/Detox-android"
```

Comment:
Surprised by this path change, why does it install in `packages/mobile/node_modules` now?

Reply:
I am actually having some trouble with detox because their device api seems to not work prioperly. Sometimes the app does not relaunch and so I tried upgrading detox to see if that fixes it but now I am getting other build problems. So, to not waste more time on this for now, I will revert the package upgrades, disable the e2e tests that don't pass and come back to it with a later PR. It would be really good to have e2e tests for this so it doesn't break in the future but for now I am going faster to fix more P1s.